misc/python/materialize/test_analytics/connector/test_analytics_connector.py

```diff
@@ -127,7 +127,7 @@ def submit_update_statements(self) -> None:
         if len(self.update_statements) == 0:
             return
 
-        cursor = self.create_cursor(autocommit=not self._use_transaction)
+        cursor = self.create_cursor(autocommit=not self._use_transaction, statement_timeout="2s")
```

Comment:
Doubled from originally `1s` but we can also go higher... (`5s`?)

Reply:
Go to 5s, or even 30, I think that's fine.

Reply:
Updates are known to have horrible performance, so this is totally expected